cinema/views.py

```diff
@@ -1 +1,73 @@
-# write views here
+from rest_framework import viewsets
+
+from cinema.serializers import (
+    ActorSerializer,
+    GenreSerializer,
+    CinemaHallSerializer,
+    MovieSerializer,
+    MovieSessionSerializer,
+    MovieListSerializer,
+    MovieRetrieveSerializer,
+    MovieSessionListSerializer,
+    MovieSessionRetrieveSerializer
+)
+from .models import (
+    Actor,
+    Genre,
+    CinemaHall,
+    Movie,
+    MovieSession
+)
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+
+class GenreViewSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects
```

Comment:
The `queryset` for `MovieViewSet` is incomplete. It should be defined as `Movie.objects.all()` to ensure it returns all movie objects.